### DISCUSSION TITLE
Please upgrade Node version

### DISCUSSION THREAD

**Discussion by igorschoester** [woocommerce/woocommerce#44601] [WooCommerce New Product Editor]
When trying to install `@woocommerce/block-templates`, I get the error:

> error @woocommerce/expression-evaluation@1.0.0: The engine "node" is incompatible with this module. Expected version "^16.14.1". Got "18.19.0"

When trying to install `@woocommerce/product-editor`, I get the error:

> error @woocommerce/components@12.2.0: The engine "node" is incompatible with this module. Expected version "^16.14.1". Got "18.19.0"

It would be nice to be able to add these packages to our setup, so we can test with the code provided in the WordPress environment.

**Comment by mattsherman** [comment]
We currently suggest using NVM (from [Woo monorepo README](https://github.com/woocommerce/woocommerce/blob/trunk/README.md)):

> [NVM](https://github.com/nvm-sh/nvm#installing-and-updating): While you can always install Node through other means, we recommend using NVM to ensure you're aligned with the version used by our development teams. Our repository contains [an .nvmrc file](https://github.com/woocommerce/woocommerce/blob/trunk/.nvmrc) which helps ensure you are using the correct version of Node.

**Reply by igorschoester** [reply]
When pulling in the above dependencies from the NPM repository. I get the mentioned errors due to the specified `engines.node` inside the `package.json` being below the Node version we use. I just wanted to use the built version. I would think I don't even need Node for that?
